example/_433D/_433.js

```diff
@@ -0,0 +1,317 @@
+let pi;
```

Comment:
_433.js is a very complex example. I'm not sure if many people will try to understand it. I currently don't have a way to verify if functions correctly. It's also a lot of code to maintain in the future. I'd prefer if it was removed. The same applies to start_433D.js.